src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+  body {
+  margin: 0 auto;
+  font-family: Roboto, sans-serif;
+  }
+
+  .header {
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  background-color: #fff;
+  display: flex;
+  justify-content:space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  max-width: 1200px;
+  }
+
+  .logo_img {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  }
+
+  .nav_list {
   margin: 0;
-}
+  padding: 0;
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  }
+
+  .nav_item {
+  margin: 0 0 0 20px;
+  }
+
+  .nav_link {
+  position: relative;
+  display: block;
+  line-height: 60px;
```

Comment:
set correct line-height from figma

Reply:
I calculated line-heidht: hight of element (logo) is 40 px and + 10 px top and bottom